### PR TITLE
Fix french setup broken links

### PR DESCRIPTION
**Pull request by thcdrt**
Hello,

Browsing the french documentation I found some broken links on the following pages:

- https://kubernetes.io/fr/docs/setup/#a-suivre
- https://kubernetes.io/fr/docs/setup/pick-right-solution/#tableau-des-solutions

Here is the patch.

Thomas

### CODE REVIEW

**Comment by thcdrt** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)
> 
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md). If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

CLA should be ok now.

**Comment by RA489** [comment]
@thcdrt Do you have any issue while signing the cla?

**Comment by thcdrt** [comment]
CLA must be ok now. I think I didn't do it right last time.

**Comment by thcdrt** [comment]
@mhamdisemah well I signed everything, I don't understand why the CLA is not good.

File: content/fr/docs/setup/_index.md [hunk 1]
```diff
@@ -76,18 +76,18 @@ Vous devriez choisir une solution de cloud clé en main sur site si vous :
...
 différents systèmes d'exploitation.
 
-Choisissez une [solution personnalisée] (/docs/setup/pick-right-solution/#custom-solutions).
+Choisissez une [solution personnalisée] (/fr/docs/setup/pick-right-solution/#solutions-personnalisées).
 
 {{% /capture %}}
 
 {{% capture whatsnext %}}
-Allez à [Choisir la bonne solution] (/docs/setup/pick-right-solution/) pour une liste complète de solutions.
+Allez à [Choisir la bonne solution] (/fr/docs/setup/pick-right-solution/) pour une list complète de solutions.
```
Comment (feloy) [inline content/fr/docs/setup/_index.md]: une liste

Comment (thcdrt) [reply content/fr/docs/setup/_index.md]: En effet, merci !

Comment (thcdrt) [reply content/fr/docs/setup/_index.md]: Fixed

**Pull request merged**